spec/rubocop/cop/mable/no_post_in_graph_ql_spec.rb

```diff
@@ -17,7 +17,21 @@
 
     context 'with query and path as helper' do
       let(:code) { "RSpec.describe 'GraphQL' do #{offense_method} end" }
-      let(:offense_method) { "post '/graphql', params: { query: graphql_query }" }
+      let(:offense_method) { "post graphql_path, params: { query: graphql_query }" }
```

Comment:
Actual changes here. I also fiddled with wording to make rubpcop a little happier...